tests/generic/models.py

```diff
@@ -231,6 +231,16 @@ class Classroom(models.Model):
     active = models.BooleanField(null=True)
 
 
+class ClassroomM2MRelated(models.Model):
+    """
+    This model was created in order to reproduce the scenario described
+    at issue 248 that is: a model with a M2M field (Classroom) being also used
+    as a M2M field from another model (ClassroomM2MRelated)
+    """
+
+    related_classrooms = models.ManyToManyField(Classroom)
```

Comment:
Just by adding this model I was able to break 2 tests with the same error message as described by @a-edakin in #248. Here's the traceback of the failing tests to prove it:

```
______________________________________________________________ TestFillNullsTestCase.test_create_nullable_many_to_many_if_flagged_and_fill_optional _______________________________________________________________
tests/test_baker.py:621: in test_create_nullable_many_to_many_if_flagged_and_fill_optional
    classroom = baker.make(models.Classroom, make_m2m=True, _fill_optional=True)
model_bakery/baker.py:133: in make
    return baker.make(
model_bakery/baker.py:389: in make
    return self._make(**params)
model_bakery/baker.py:457: in _make
    self.model_attrs[field.name] = self.generate_value(
model_bakery/baker.py:660: in generate_value
    if field.has_default() and field.name not in self.rel_fields:
E   AttributeError: 'ManyToManyRel' object has no attribute 'has_default'
________________________________________________________ TestFillNullsTestCase.test_nullable_many_to_many_is_not_created_if_not_flagged_and_fill_optional _________________________________________________________
tests/test_baker.py:627: in test_nullable_many_to_many_is_not_created_if_not_flagged_and_fill_optional
    classroom = baker.make(models.Classroom, make_m2m=False, _fill_optional=True)
model_bakery/baker.py:133: in make
    return baker.make(
model_bakery/baker.py:389: in make
    return self._make(**params)
model_bakery/baker.py:457: in _make
    self.model_attrs[field.name] = self.generate_value(
model_bakery/baker.py:660: in generate_value
    if field.has_default() and field.name not in self.rel_fields:
E   AttributeError: 'ManyToManyRel' object has no attribute 'has_default'
```